client/src/main/java/org/jboss/fuse/mvnd/client/DaemonConnector.java

```diff
@@ -362,7 +362,7 @@ public boolean maybeStaleAddress(Exception failure) {
     }
 
     public DaemonConnection<Message> connect(int port) throws DaemonException.ConnectException {
-        InetSocketAddress address = new InetSocketAddress(port);
+        InetSocketAddress address = new InetSocketAddress(Inet4Address.getLoopbackAddress(), port);
```

Comment:
Import seems to missing (spotted this while trying out this commit in my fork).

I also wonder why not `InetAddress` instead of `Inet4Address`?